src/NuGet/Microsoft.CodeAnalysis.CSharp.Workspaces.nuspec

```diff
@@ -27,6 +27,8 @@ Supported Platforms:
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\portable-net45+win8" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\net45" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\net45" />
+    <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\netstandard1.3" />
```

Comment:
You can depend on the common dependencies coming from your common package but if there are more unique dependencies I would keep them on the specific packages that require them instead of putting them on the common package to prevent them from being pulled in for all your packages.